.gitignore

```diff
@@ -11,3 +11,5 @@
 /test/out
 
 /test/work
+
+/metasv/run_after_age.py
```

Comment:
I made that script for my own test when I ran age alone. The run_after_age.py basically do the rest after we have age output. So, do you mean I should include that in the package? because if I remove that line when I commit it also will be added.